src/com/lftechnology/java_training/jan7/RenameAndDeleteOrgFile.java

```diff
@@ -0,0 +1,116 @@
+package com.lftechnology.java_training.jan7;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.logging.Level;
+
+/**
+ * Write a program to rename the given file, after renaming the file delete the renamed file. (Accept the file name using command line
+ * arguments.)
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class RenameAndDeleteOrgFile {
+	private static Scanner input = new Scanner(System.in);
+	private static TrainingConstantsUtility constantValues = new TrainingConstantsUtility();
+
+	public static void main(String[] args) {
+		String path = "secondprogramfile.txt";
+		File newFile = new File(path);
+		createFile(newFile);
+		newFile = renameFile(newFile);
+		deleteFile(newFile);
+	}
+
+	/**
+	 * creates new file and checks if it already exists or not
+	 * 
+	 * @param newFile
+	 */
+	private static void createFile(File newFile) {
```

Comment:
Also, it is better to return Boolean of the created File so that the calling method knows if it is created or not.